test/gateway/eip20_cogateway/progress_revert_redeem.js

```diff
@@ -0,0 +1,385 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+
+const EIP20CoGateway = artifacts.require("TestEIP20CoGateway");
+const MockUtilityToken = artifacts.require("MockUtilityToken");
+const BN = require("bn.js");
+
+const messageBus = require('../../test_lib/message_bus.js');
+const Utils = require("../../test_lib/utils.js");
+const web3 = require("../../test_lib/web3.js");
+const EventDecoder = require("../../test_lib/event_decoder");
+const coGatewayUtils = require('./helpers/co_gateway_utils.js');
+const StubData = require("../../../test/data/redeem_revoked_1.json");
+
+const ZeroBytes = Utils.ZERO_BYTES32;
+const MessageStatusEnum = messageBus.MessageStatusEnum;
+
+contract('EIP20CoGateway.progressRevertRedeem() ', function (accounts) {
+
+  let utilityToken, cogateway, revertRedeemParams, bountyAmount, penaltyAmount,
+    owner,
+    facilitator;
```

Comment:
This is a really confusing way of aligning the variables. Either one per line, or all in one line.